rc/tools/ctags.kak

```diff
@@ -121,3 +140,28 @@ define-command ctags-update-tags -docstring 'Update tags for the given file' %{
         printf %s\\n "evaluate-commands -client $kak_client echo -markup '{Information}${msg}'" | kak -p ${kak_session}
     ) > /dev/null 2>&1 < /dev/null & }
 }
+
+define-command ctags-enable-autocomplete -docstring "Enable automatic ctags completion" %{
+    evaluate-commands %sh{
```

Comment:
Not sure that we need to check that, I think ctagsfiles should be tolerant to absent files, its more a set of files to look for, missing ones should not be an error.